src/proto/beats.proto

```diff
@@ -10,22 +11,46 @@ enum Source {
   USER = 2;
 }
 
-message Beat {
-  optional int32 frame_position = 1;
-  optional bool enabled = 2 [ default = true ];
-  optional Source source = 3 [ default = ANALYZER ];
+enum Type {
+  BEAT = 0;
+  BAR = 1;
+  PHRASE = 2;
+}
```

Comment:
Can we have a section type too?

Reply:
We should only define the types which we actually use now. We can add more at any time when needed without breaking backwards compatibility.

The other way round is not possible, i.e. you cannot delete types from this enum once you released them without breaking backwards compatibility.

Reply:
How about we only add BEAT and BAR for now? Then @crisclacerda can modify #2877 to output the detected downbeats to this new protobuf format. When @crisclacerda implements phrase and section detection we can add those, or alternatively if @hacksdump implements the UI for editing phrases and sections before the analysis is implemented he could add those to the protobuf at that time.

Reply:
Exactly. If we don't use PHRASE yet, don't add it.